### PR TITLE
i18n(fr) Update `github.mdx`

### PR DESCRIPTION
**Pull request by McFlyPartages**
Update french translation with PR #5572 #5635 #5870

#### Related issues & labels (optional)

- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/fr/guides/deploy/github.mdx [hunk 1]
```diff
@@ -13,36 +13,77 @@ Vous pouvez déployer un site Astro sur les GitHub Pages en utilisant une [GitHu
 
 Astro maintient l'action officielle `withastro/action` pour déployer vos projets avec très peu de configuration. Suivez les instructions ci-dessous pour déployer votre site Astro sur GitHub Pages, et consultez [le README du GitHub Action Astro](https://github.com/withastro/action) pour plus d'informations.
 
-1. Définissez les options [`site`](/fr/reference/configuration-reference/#site), et si necessaire, [`base`](/fr/reference/configuration-reference/#base) dans `astro.config.mjs`.
+## Configurer Astro pour les pages GitHub
 
-    ```js title="astro.config.mjs" ins={4-5}
-    import { defineConfig } from 'astro/config'
+### Déployer vers une URL `github.io
```
Comment (thomasbnt) [inline src/content/docs/fr/guides/deploy/github.mdx:18]:
```suggestion
### Déployer vers une URL `github.io`
```

File: src/content/docs/fr/guides/deploy/github.mdx [hunk 2]
```diff
@@ -13,36 +13,77 @@ Vous pouvez déployer un site Astro sur les GitHub Pages en utilisant une [GitHu
...
-    ```astro
-    <a href="/my-repo/about">À propos</a>
-    ```
-    :::
+#### `site`
+
+The value for `site` must be one of the following:
+
+- The following URL based on your username: `https://<username>.github.io`
+- The random URL autogenerated for a [GitHub Organization's private page](https://docs.github.com/en/enterprise-cloud@latest/pages/getting-started-with-github-pages/changing-the-visibility-of-your-github-pages-site): `https://<random-string>.pages.github.io/`
```
Comment (thomasbnt) [inline src/content/docs/fr/guides/deploy/github.mdx:36]:
```suggestion
La valeur de `site` doit être l'une des suivantes :

- L'URL suivante en fonction de votre nom d'utilisateur : `https://<username>.github.io`
- L'URL aléatoire autogénérée pour une [page privée de l'organisation GitHub](https://docs.github.com/en/enterprise-cloud@latest/pages/getting-started-with-github-pages/changing-the-visibility-of-your-github-pages-site) : `https://<random-string>.pages.github.io/`
```

**Review by thomasbnt — CHANGES_REQUESTED** [review]
Some phrases aren't translated and one " ` " missed. The rest is LGTM

**Comment by McFlyPartages** [comment]
> Some phrases aren't translated and one " ` " missed. The rest is LGTM

Oups; Thanks

**Review by thomasbnt — APPROVED** [review]
LGTM

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🙌

**Pull request merged**